src/test/groovy/com/spidasoftware/schema/conversion/changeset/v10/IceDensityChangeSetTest.groovy

```diff
@@ -156,6 +156,12 @@ class IceDensityChangeSetTest extends Specification {
             !json.leads[0].locations[0].designs[0].clearanceCases[1].upperPhysicalState.wireState.designCondition.iceDensity
             !json.leads[0].locations[0].designs[0].clearanceCases[1].upperPhysicalState.wireState.highestStressCondition.iceDensity
             !json.leads[0].locations[0].designs[0].clearanceCases[1].upperPhysicalState.wireState.creepCondition.iceDensity
+            !json.leads[0].locations[0].designs[0].clearanceCases[2].upperThermalState.wireState.designCondition.iceDensity
+            !json.leads[0].locations[0].designs[0].clearanceCases[2].upperThermalState.wireState.highestStressCondition.iceDensity
+            !json.leads[0].locations[0].designs[0].clearanceCases[2].upperThermalState.wireState.creepCondition.iceDensity
+            !json.leads[0].locations[0].designs[0].clearanceCases[2].upperPhysicalState.wireState.designCondition.iceDensity
+            !json.leads[0].locations[0].designs[0].clearanceCases[2].upperPhysicalState.wireState.highestStressCondition.iceDensity
```

Comment:
these new tests should confirm that `lowerPhysicalState` gets iceDensity added to its weather conditions